docs/extras/autorcm.md

```diff
@@ -1,17 +1,22 @@
-### **AutoRCM**
+# AutoRCM
 
 AutoRCM causes the console to believe it is bricked, and will automatically launch RCM upon boot for recovery purposes, without needing a jig. As RCM is a recovery mode from repair specialists, this is an intended feature from the device developer, though is also considered a softbrick. If you aren't careful, misuse of AutoRCM can lead to real **damage**, especially with units that cannot inject custom RCM payloads (like Mariko hardware). Please take care when using it. Please keep in mind that the console can no longer boot on its own, so you'll need a PC, phone, or other payload injector to start the console after a coldboot.
 
 - Note: If the console screen stays black when pressing the power button after activating AutoRCM, remember that it's not broken. It's in RCM.
 
-!!! warning "If you don't have a BOOT0/1 backup yet..."
-    You really want to kill your console, huh? If you haven't made a BOOT0/1 backup yet, it is recommended to make one **right now**.
+::: warning
 
-    1. Boot into Hekate
-    1. Go to `Tools`, and select `Backup eMMC`
-    1. Then tap `eMMC BOOT0 & BOOT1` and let the process complete.
+**If you don't have a BOOT0/1 backup yet...**
 
-### **Disadvantages of AutoRCM**
+You really want to kill your console, huh? If you haven't made a BOOT0/1 backup yet, it is recommended to make one **right now**.
```

Comment:
Unrelated to this PR, but we may want to check for such snarky comments in the future, and remove them.